pkg/deploy/util/util.go

```diff
@@ -390,6 +397,35 @@ func int32AnnotationFor(obj runtime.Object, key string) (int32, bool) {
 	return int32(i), true
 }
 
+// DeploymentsForCleanup determines which deployments for a configuration are relevant for the
+// revision history limit quota
+func DeploymentsForCleanup(configuration *deployapi.DeploymentConfig, deployments []api.ReplicationController) []api.ReplicationController {
```

Comment:
It's also called in the test, it made a lot of sense to ensure that we were really cleaning up as we wanted to be *e.g.* by sharing the code. I'm not sure what you mean by the zero check